container/lxd/initialisation.go

```diff
@@ -122,19 +122,36 @@ func configureLXDProxies(setter configSetter, proxies proxy.Settings) error {
 	return nil
 }
 
-var execCommand = exec.Command
+// df returns the number of free bytes on the file system at the given path
+var df = func(path string) (uint64, error) {
+	statfs := unix.Statfs_t{}
+	err := unix.Statfs(path, &statfs)
+	if err != nil {
+		return 0, err
+	}
+
+	return uint64(statfs.Bsize) * statfs.Bfree, nil
+}
 
 var configureZFS = func() {
```

Comment:
if you are already touching the code you could also modify this into a function.